static/app/views/insights/http/views/httpDomainSummaryPage.tsx

```diff
@@ -22,14 +18,15 @@ import {ModulePageProviders} from 'sentry/views/insights/common/components/modul
 import {ModuleBodyUpsellHook} from 'sentry/views/insights/common/components/moduleUpsellHookWrapper';
 import {ReadoutRibbon, ToolRibbon} from 'sentry/views/insights/common/components/ribbon';
 import {getTimeSpentExplanation} from 'sentry/views/insights/common/components/tableCells/timeSpentCell';
+import {useHttpDomainSummaryFilter} from 'sentry/views/insights/common/components/widgets/hooks/useHttpDomainSummaryFilter';
+import HttpDomainSummaryDurationWidget from 'sentry/views/insights/common/components/widgets/httpDomainSummaryDurationWidget';
```

Comment:
See https://github.com/getsentry/sentry/pull/89045/files#diff-886e46c8485496b33137c9770f4eb4fb6e1085c7e6e0fbafdd5721d4310b74e3R33-R62 on how we're going to use these widgets -- these are default exports to make it easier to dynamically import and render.